MicrosoftFluentUI.podspec

```diff
@@ -326,7 +326,7 @@ xcodebuild ${XCODEBUILDPARAMS} -project ${PROJECT_FILE_PATH} -target "MicrosoftF
   s.subspec 'Core_mac' do |core_mac|
     core_mac.platform = :osx
     core_mac.resource_bundle = { 'FluentUIResources-macos' => ["apple/Resources/**/*.{json,xcassets}",
-                                                             "macos/FluentUIResources-macos/**/*.{storyboard,xib,xcassets,strings,stringsdict}"] }
+                                                             "macos/FluentUI/**/*.{storyboard,xib,xcassets,strings,stringsdict}"] }
```

Comment:
i see so macoS we need to update the podspec but not for ios?

Reply:
Correct! iOS was already set up the way it needed to be. Seems like someone moved it in the past?